scenarioo-client/test/e2e/pages/detailAreaPage.ts

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+import {by, element} from 'protractor';
+
+class DetailAreaPage {
+
+    private toggleMetaDataButton = element(by.id('sc-showHideDetailsButton'));
+    private metaDataPanel = element(by.id('sc-metadata-panel'));
+
+    async assertDetailsAreaExpanded() {
+        await expect(this.toggleMetaDataButton.getText()).toEqual('Hide details');
+        return expect(this.metaDataPanel.isDisplayed()).toBe(true);
+    }
+
+    async assertDetailsAreaCollapsed() {
+        await expect(this.toggleMetaDataButton.getText()).toEqual('Show details');
+        return expect(this.metaDataPanel.isDisplayed()).toBe(false);
+    }
+
+    async expandDetailsArea() {
+        return this.toggleMetaDataButton.click();
+    }
+
+    async collapseDetailsArea() {
+        return this.toggleMetaDataButton.click();
+    }
+
+    async assertSectionExpanded(detailsSection) {
+        const section = element(by.id('sc-section-' + detailsSection));
+        return expect(section.getAttribute('aria-expanded')).toEqual('true');
+    }
+
+    async assertSectionCollapsed(detailsSection) {
+        const section = element(by.id('sc-section-' + detailsSection));
+        return expect(section.getAttribute('aria-expanded')).toEqual('false');
+    }
+
+    async expandDetailsSection(detailsSection) {
+        const toggleSection = element(by.id('sc-toggleSection-' + detailsSection));
+        return toggleSection.click();
+    }
+
+    async collapseDetailsSection(detailsSection) {
+        const toggleSection = element(by.id('sc-toggleSection-' + detailsSection));
+        return toggleSection.click();
+    }
```

Comment:
if you provide two methods for expand and collapsed I would expect that they not both do the same -- so at least check the expected outcome and/or precondition?